tools/clang/lib/Sema/SemaHLSL.cpp

```diff
@@ -12888,7 +12879,6 @@ void Sema::TransferUnusualAttributes(Declarator &D, NamedDecl *NewDecl) {
   if (!getLangOpts().HLSL) {
     return;
   }
-
```

Comment:
Why one deleted blank line here?

Reply:
Fixed. It is caused revert local change.